yaksh/models.py

```diff
@@ -879,6 +879,7 @@ def create_duplicate_course(self, user):
             copy_module_name = "Copy of {0}".format(module.name)
             new_module = module._create_module_copy(user, copy_module_name)
             new_course.learning_module.add(new_module)
+        return new_course
```

Comment:
Why not `return new_course.learning_module.add(new_module)`

Reply:
Will not work. I need to return the new course.